src/actions/targetActions.js

```diff
@@ -18,6 +18,22 @@ export const addTargetFailed = errors => ({
   type: actions.ADD_TARGET_FAILED, errors
 });
 
+export const selectTargetSuccess = target => ({
```

Comment:
The action creators that end with `Success` or `Failed` should be those that go to the API where something can fail.

These should not include that in the name. `selectTarget` is just fine.

Reply:
it's done